src/Connection.ts

```diff
@@ -238,8 +238,9 @@ class Connection {
         const cleanedJson = json.replace(/"(?:(?=(\\?))\1.)*?"/gs, substring => {
             return substring.replace(/(?<!\\)((?:\\\\)*)\n/g, '$1\\n');
         });
+        const tabCleanedJson = cleanedJson.replace(/\t/g, '\\t');
```

Comment:
Do you have an example response coming from FileMaker where this is an issue in the parsing process?

If it is, it should be handled in the regexp above to avoid escaping tabs which are outside of JSON strings.